saga/seata-saga-spring/src/main/java/org/apache/seata/saga/engine/store/db/DbAndReportTcStateLogStore.java

```diff
@@ -733,7 +733,7 @@ public void clearUp() {
         RootContext.unbind();
         RootContext.unbindBranchType();
         if (sagaTransactionalTemplate != null) {
-            sagaTransactionalTemplate.cleanUp();
+            sagaTransactionalTemplate.cleanUp(null);
```

Comment:
Since the globalTransaction is passed above, theoretically, they should all go to it and pass the globalTransaction